.github/workflows/promptflow-sdk-cli-test.yml

```diff
@@ -2,10 +2,11 @@ name: promptflow-sdk-cli-test
 on:
   schedule:
     - cron: "40 18 * * *" # Every day starting at 2:40 BJT
-  pull_request_target:
+  pull_request:
     paths:
       - src/promptflow/**
-      - scripts/building/**
+      - scripts/building/run_coverage_tests.py
```

Comment:
maybe we can keep `scripts/building/**`?

Reply:
Good catch